examples/cassandra/v1beta3/cassandra-controller.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata: 
+  labels: 
+    name: cassandra
+  name: cassandra
+spec: 
+  replicas: 1
+  selector: 
+    name: cassandra
+  template: 
+    metadata: 
+      labels: 
+        name: cassandra
+    spec: 
+      containers: 
+        - command: 
+            - /run.sh
+          cpu: 1000
```

Comment:
Actually, could you change this (and the previous example that I already merged) to use the new resource specification -- resources?

@vishh can answer any questions.